content/en/real_user_monitoring/browser/advanced_configuration.md

```diff
@@ -19,6 +19,59 @@ further_reading:
 
 Find below the different initialization options available with the [Datadog Browser SDK][1].
 
+### Identify user sessions
+Adding user information to your RUM sessions makes it easy to:
+* Follow the journey of a given user
+* Know which users are the most impacted by errors
+* Monitor performance for your most important users
+
+{{< img src="real_user_monitoring/browser/advanced_configuration/user-api.png" alt="User API in RUM UI"  >}}
+
+The following attributes are **optional** but it is recommended to provide **at least one** of them:
+
+| Attribute  | Description                                                                                              |
+|------------|----------------------------------------------------------------------------------------------------------|
+| usr.id    | Unique user identifier.                                                                                  |
```

Comment:
Maybe we should add a note for the fact that it will be indexed as string

Reply:
Added `Type` column